packages/test.sh

```diff
@@ -53,8 +55,8 @@ for ((i=0; i<${#PACKAGE_DIRS[@]}; i++)); do
     pip install -r test-requirements.txt
 
     # Prevent execution of alembic/env.py at test collection stage (alembic.context not set)
-    # Also ignore ToolShed functional tests.
-    unit_extra='--doctest-modules --ignore galaxy/model/migrations/alembic/ --ignore=tool_shed/test/'
+    # Also ignore functional tests (galaxy_test and tool_shed/test/).
+    unit_extra='--doctest-modules --ignore=galaxy/model/migrations/alembic/ --ignore=tool_shed/test/  --ignore=galaxy_test/api --ignore=galaxy_test/selenium'
```

Comment:
There needs to be at least one unit or doc test per package, or pytest will fail below. This is why I added one doc test to both test_base and test_api inside their `galaxy_test/` .

Reply:
OK, I moved the doctests to unit tests and excluded the whole `galaxy_test/` again here. We just need to add some unit tests to `test_api` and `test_selenium` now.

Reply:
It feels silly right?

Reply:
> It feels silly right?

If it doesn't have tests, it's probably broken 😄

> Update: Should we catch the pytest exit code 5 (no tests run) and suppress it somehow?

Yeah, that should be possible and a good workaround. Let me know if you'd like help with that.

Reply:
> If it doesn't have tests, it's probably broken 😄

They're tests 😆... but are the packaged tests broken... yet 😆.